internal/dag/policy.go

```diff
@@ -35,7 +35,7 @@ func retryPolicy(rp *projcontour.RetryPolicy) *RetryPolicy {
 
 // ingressRetryPolicy builds a RetryPolicy from ingress annotations.
 func ingressRetryPolicy(ingress *v1beta1.Ingress) *RetryPolicy {
-	retryOn, ok := ingress.Annotations[annotationRetryOn]
+	retryOn, ok := ingress.Annotations["projectcontour.io/retry-on"]
```

Comment:
Here, you can use `compatAnnotation(ingress, "retry-on")` to handle falling back from the contour annotation to the heptio one.

Reply:
Maybe update `ingressTimeoutPolicy` below while you are here?